src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -143,7 +116,21 @@ module.exports = themeConfig;
 
 Toggle the theme from the dropdown at the top of the page to see changes. The full example code can be seen [here](https://github.com/cerner/terra-application/tree/master/src/terra-dev-site/doc/application/how-to.2/example).
 
-<ExampleComponent />
+<ExampleComponent isExpanded />
+
+## Testing
```

Comment:
Should we offer any jest testing advice?

Reply:
I think we should circle back around once we've settle on a strategy and exposed a theme provider from terra-application. I'm personally team react-testing-library over enzyme when it comes to testing hooks and context, but I don't think we should start recommending that to people until we've discussed it a little more.